src/sass/blog/_blog-interstitial.scss

```diff
@@ -29,14 +29,44 @@
 	}
 
 	/** Subscription Form */
+
+	form {
+		display: table;
+		width: 100%;
+	}
+	label {
+		display: inline-block;
+		width: 70%;
+
+		@include breakpoint( bp() ) {
+			display: table-cell;
+			width: 100%;
+		}
+	}
+	input[type="email"] {
+		@include breakpoint( bp() ) {
+			display: table-cell;
+		}
+	}
 	input[type="email"] {
 		@extend %roboto-normal;
-		display: inline-block;
+		display: block;
 		font-size: 1rem;
 		line-height: 1rem;
-		margin-left: 1em;
-		margin-right: 0;
+		margin-right: 0.2em;
```

Comment:
Since we're going with `display: table;` for this piece, I would take the margins away from the input element and apply them to the form element instead.